src/server/transport/socketio.js

```diff
@@ -62,6 +62,7 @@ export function SocketIO({
   clientInfo = new Map(),
   roomInfo = new Map(),
   auth = true,
+  https = undefined,
```

Comment:
This can just be:

```js
https,
```

The default value will always be undefined.

Reply:
Done.